projects/mongo-core-test.yml

```
@@ -0,0 +1,26 @@
+---
+runner: command
+params:
+  dir: mongo-core-test
+  setup: |
+    python3 -m venv .
+    COMPOSE_FILE='mongo5.yml' TEST_DIRECTORY=$(readlink -f ../mongo/jstests) docker compose up -d
```

```suggestion
    env COMPOSE_FILE='mongo5.yml' TEST_DIRECTORY=$(readlink -f ../mongo/jstests) docker compose up -d legacy-mongo
```

There is no need to start all services.